demo/src/main/java/standalone/MarkerController.java

```diff
@@ -111,9 +111,19 @@ public MarkerController() {
                 )
             )
             .put("origin-intersecting-curve",
-                    getSolution(
-                    "through: bottomLeft, origin, topRight"
-                    )
+                getSolution(
+                "through: bottomLeft, origin, topRight"
+                )
+            )
+            .put("the-problem",
```

Comment:
:eyes:

Reply:
Ah I left the temporary name in, I think it is still worthwhile to include this problem in the sample set as there are none currently that have multiple intersections. Will do another push with the name change, I also realise that I forgot to include the sample files